### PR TITLE
Event-182

### PR DESCRIPTION
**Pull request by adammeyer**
Plus permission checks for actions

### CODE REVIEW

**Comment by adammeyer** [comment]
@ryancarlson this is for the new scholarship permission.

File: app/scripts/controllers/paymentModal.js [hunk 1]
```diff
@@ -39,6 +40,18 @@ angular.module('confRegistrationWebApp')
         return;
       }
 
+      if(permissions.permissionInt < permissionConstants.UPDATE){
+        if(permissions.permissionInt === permissionConstants.SCHOLARSHIP) {
+          if($scope.newTransaction.paymentType !== 'SCHOLARSHIP'){
+            alert('Your permission level only allows scholarship payments to be added.');
```
Comment (ryancarlson) [inline app/scripts/controllers/paymentModal.js]: Maybe add this phrase as part of your alert here too? 
"Please contact an event admin to request permission."

File: app/views/permissionError.html [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <div class="alert alert-danger">
-  Sorry, you do not have permission to access this page.  Please contact the event creator to update your permissions.
-</div>
\ No newline at end of file
+  Sorry, you do not have permission to access this page.  Please contact an event admin to request permission.
```
Comment (ryancarlson) [inline app/views/permissionError.html]: "admininstrator" instead of "admin"

File: app/scripts/controllers/paymentModal.js [hunk 3]
```diff
@@ -12,6 +12,7 @@ angular.module('confRegistrationWebApp')
       amount: 0,
       sendEmailReceipt: false
     };
+    var permissionRequiredMsg = 'You do not have permission to perform this action. Please contact an event admin to request permission.';
```
Comment (ryancarlson) [inline app/scripts/controllers/paymentModal.js]: "administrator" here too.

**Comment by ryancarlson** [comment]
Other than the minor wording changes.. :+1:  nice job!  I love the permission level comparison chart.  Great idea

**Pull request merged**